virttest/utils_misc.py

```diff
@@ -3045,6 +3045,79 @@ def get_windows_drive_letters(session):
     return drive_letters
 
 
+class IozoneTest(object):
```

Comment:
I would suggest to have a separate file utils_iozone.

Reply:
How about move it to utils_test? Since I need to remove iozone test from utils_test.VMStress, its load_stress() function will lose iozone part there. Refer to https://github.com/avocado-framework/avocado-vt/pull/1467

Reply:
@vivianQizhu, Agree with your suggestion it can be placed in utils_test. Also in utils_test can we have a common class that would have code for setup/install, check(), cleanup/uninstall() part where IozoneTest/VMStress can avoid redundancy by reusing it ?

Reply:
@balamuruhans Yes, these two have similar functions, I will check with @aliang123 to see how we can reorganise it.

Reply:
sure :+1:

Reply:
Discussed with vivianQizhu, it will be realized in utils_test.VMStress. So will close this patch, if the new patch are available, thanks for reviewing.